modules/cli/src/integrationTest/java/org/apache/ignite/cli/commands/status/ItStatusReplCommandTest.java

```diff
@@ -0,0 +1,44 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.ignite.cli.commands.status;
+
+import static org.junit.jupiter.api.Assertions.assertAll;
+
+import org.apache.ignite.cli.commands.CliCommandTestIntegrationBase;
+import org.junit.jupiter.api.Disabled;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+/**
+ * Tests for {@link StatusReplCommand}.
+ */
+class ItStatusReplCommandTest extends CliCommandTestIntegrationBase {
+
+    @Test
+    @DisplayName("Should print status when valid cluster url is given")
+    @Disabled // TODO: https://issues.apache.org/jira/browse/IGNITE-17091
```

Comment:
`@Disabled` has the special form with the reason of disable `@Disabled(https://issues.apache.org/jira/browse/IGNITE-17091)`. It will be better, than sepparate comment, I think. Use it here and in the other disables from PR, pls, if you don't mind.